src/include/binder/binder_context.h

```diff
@@ -13,6 +13,7 @@
 #include <unordered_map>
 #include <string>
 #include "type/types.h"
+#include "concurrency/transaction.h"
```

Comment:
Forward declare the transaction class and remove the include, since the definition isn't needed in the header.